src/style.css

```diff
@@ -1 +1,79 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Book.ttf'), url('fonts/Avenir-Heavy.ttf');
+  font-weight: 300;
+}
+
+body {
+  margin: 0;
+}
+
+.search__bar {
+  margin: 20px 8px 0;
+}
+
+.search__bar--input {
+  width: 100%;
+  font-family: Avenir, sans-serif;
+  color: #3d4e61;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+}
+
+.search__bar--input::placeholder {
+  color: #3d4e61;
+}
+
+.input__big {
+  height: 70px;
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 21.86px;
+  border-radius: 3px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.input__big:hover {
+  box-shadow: 0 3px 8px 0 rgba(61, 78, 97, 0.2);
+}
+
+.input__big:focus {
+  font-family: AvenirHeavy, sans-serif;
+  text-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
```

Comment:
Avoid using text-shadow for focus states as it can make text harder to read. Consider using other styles to indicate focus, such as border or outline.